django/core/files/move.py

```diff
@@ -66,7 +66,13 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
+        # This will happen with copystat if the destination is a CIFS mount
+        if getattr(e, 'errno', 0) != 1:
```

Comment:
FYI, `errno.EPERM` maps to errno 1 so this is the errno we will check for instead of an integer like we originally submitted in the PR.